charters/Architecture/charter.md

```diff
@@ -0,0 +1,48 @@
+# DCP Architecture Team
+
+## Description
+
+The Data Coordination Platform (DCP) Architecture Team is responsible for the technical oversight of the DCP, including
+identifying and addressing *cross-cutting* architecture, integration, and operational concerns across DCP
+projects.
+
+The core team is composed of *Technical Leads* representing each technical DCP project group to ensure a diversity of
+skills and views needed for community technical decision-making. 
+
+## In-scope
+
+Software architecture responsibilities
+* Set overall direction of architectural and software design direction for the DCP
+* Provide technical input for the PM product roadmap process
+* Review and provide final approval for technical charters for DCP projects
+* Publish Architecture Requests for Comments (RFCs) documenting technical debt that impacts delivery of DCP as an
```

Comment:
I think the specifics here (technical debt, DCP as an integrated system) aren't needed. The arch team role will go beyond this quite narrow scope, and will publish RFCs for any major change. I'd suggest 
* Publish Requests for Comments (RFCs) for any significant change to the DCP architecture

Reply:
I think doing this would change the core responsibilities in the role. Currently, Arch's role is framed as advisory and mentorship, acknowledging that we will (hopefully) receive many submissions from the community.

This point is specifically about making tech debt visible, not the about the format in which we communicate desired change (RFCs).

Reply:
OK, so leading with "Publish Architecture Requests for Comments (RFCs)..." is probably what caused me to misinterpret this. I agree that we shouldn't be discouraging community contributions, I definitely was not intending to suggest the arch group was the only source of RFCs! So let's try that again... 😄 

If you are attempting to highlight the advisory role of the arch group, I'd say "documenting technical debt" is a very passive way of doing so. Who are you expecting to read this documentation? I prefer proposal of solutions rather than identification of problems.

How about something like:
 - Identify risks impacting delivery of the DCP roadmap and propose appropriate mitigation strategies by publishing requests for comments (RFCs) ...?